metacoadd/exposure.py

```diff
@@ -111,7 +97,7 @@ def __init__(
         if set_meta:
             self._set_meta()
 
-    def __getitem__(self, bounds):
+    def __getitem__(self, bounds: galsim.BoundsI) -> Self:
```

Comment:
If I'm right this should return an exposure

```suggestion
    def __getitem__(self, bounds: galsim.BoundsI) -> Exposure:
```

Reply:
Actually, Exposure is not yet defined when encountering this line, so it does not compile... The Self is a trick in python 3.11. The easiest is just to remove it (or there are alternative for previous version)

Reply:
Actually, it seems possible to write -> "Exposure" in this case. I did that.